libs/ngx-dynamic-form-builder/src/lib/utils/dynamic-form-group.ts

```diff
@@ -38,7 +42,7 @@ export class DynamicFormGroup<TModel> extends FormGroup {
         this.defaultValidatorOptions
       );
     });*/
-    this.fields = this.onlyFields(this.fields);
+    this.formFields = this.onlyFields(fields);
```

Comment:
Prevents types from being cumbersome in the code. The FormModel<T> type is to enhance the implementer's experience by helping them adhere to their types. Once in this framework code, the type safety is no longer an issue as it is all dynamic.